pulp_python/app/serializers.py

```diff
@@ -1,30 +1,87 @@
+from gettext import gettext as _
+
 from rest_framework import serializers
 from pulpcore.plugin import serializers as platform
 
 from . import models
 
 
-class PythonContentSerializer(platform.ContentSerializer):
+class ClassifierSerializer(serializers.ModelSerializer):
+    """
+    A serializer for Python Classifiers
     """
-    A Serializer for PythonContent.
 
-    Add serializers for the new fields defined in PythonContent and
-    add those fields to the Meta class keeping fields from the parent class as well.
+    name = serializers.CharField(
+        help_text=_("The name of the classifier")
+    )
 
-    For example::
+    class Meta:
+        model = models.Classifier
+        fields = ('name',)
 
-    field1 = serializers.TextField()
-    field2 = serializers.IntegerField()
-    field3 = serializers.CharField()
 
-    class Meta:
-        fields = platform.ContentSerializer.Meta.fields + ('field1', 'field2', 'field3')
-        model = models.PythonContent
+class PythonPackageContentSerializer(platform.ContentSerializer):
+    """
+    A Serializer for PythonPackageContent.
     """
 
+    filename = serializers.CharField()
+    path = serializers.CharField()
+    packagetype = serializers.CharField()
+    name = serializers.CharField()
+    version = serializers.CharField()
+    metadata_version = serializers.CharField()
+    summary = serializers.CharField(required=False, allow_blank=True)
```

Comment:
I don't want to pile on a time consuming task, but these all need help text. If you prefer, we can have decide them together on a call and do that as a separate task.